Makefile

```diff
@@ -2,6 +2,7 @@ DEV_IMAGE_ORG = jboss-dataservices
 DOCKER_REGISTRY_ENGINEERING =
 DOCKER_REGISTRY_REDHAT =
 DEV_IMAGE_NAME = datagrid-online-services-dev
+ADDITIONAL_ARGUMENTS =
```

Comment:
AFAICT this is always empty

Reply:
Yeah, but you can add additional parameters to the test if you with... e.g. `make test-functional ADDITIONAL_PARAMETERS="-DTest=CachingServiceTest"`. Sometimes this might be useful...

Reply:
O cool, I'm not too familiar with Make files so didn't realise